.github/workflows/tests.yaml

```diff
@@ -377,6 +377,13 @@ jobs:
       - name: Set CLI context
         run: astro context switch ${{ steps.get-astro-env-info.outputs.astronomer_host }}
 
+      - name: Mock git commands for DAG Deploy
+        run: |
+
+          # set to no deploy git mock, to simulate no interested file change scenario
+          mv e2e-setup/mocks/no-deploy-git.sh /usr/local/bin/git
```

Comment:
fixing a case that I found after merging my previous PR, as for the no-deploy case we want to simulate a change in a file outside the root folder. This is